lib/chezmoi/chezmoi.go

```diff
@@ -19,6 +19,7 @@ const (
 	executablePrefix = "executable_"
 	dotPrefix        = "dot_"
 	templateSuffix   = ".tmpl"
+	scriptsDir       = ".scripts"
```

Comment:
Please use separate directories for "run once" scripts and "run every time" scripts instead of checking for `chezmoi:once` regexps in the scripts.

Reply:
I really don't like having separate directories. Could you read and comment on [this](https://github.com/twpayne/chezmoi/pull/206#issuecomment-465315877) (especially the first one) maybe?

Reply:
Thanks, I'll read [this](https://github.com/twpayne/chezmoi/pull/206#issuecomment-465315877).

Please can you make the scripts directory be `.chezmoiscripts` for consistency with `.chezmoiignore`.